engines/director/detection_tables.h

```diff
@@ -2260,6 +2261,13 @@ static const DirectorGameDescription gameDescriptions[] = {
 	WINDEMO2t("wrath", "Demo", "WRATH.EXE",	 "5d0ee796571b99d402a06438ae2f3d56", 696815,
 							   "SHARED.DIR", "e709f7b88f6241e4f45632beb0533ac1", 3570542, 404),
 
+	// Jan Lindblad presenterar den sjungande Fågelboken
+	// There is also a Macintosh version which also appears to be using
+	// Macromedia 4.0.4. But dumping that version still gives you the
+	// lindblad.exe file, and I assume all the data files are functionally
+	// the same, even if they have resource and data forks.
+	WINGAME1t_l("singingbirdbook", "", "lindblad.exe", "713376834e4e94d87ad931a66c3575fc", 698995, Common::SE_SWE, 404),
+
```

Comment:
If there's a mac director executable we usually add it anyway.
- The entry should be moved after line 3337, as this is not an officially supported game
- For new entries we indicate the tail in the checksum:
`WINGAME1_l("singingbirdbook", "", "lindblad.exe", "t:713376834e4e94d87ad931a66c3575fc", 698995, Common::SE_SWE, 404),`

Reply:
Oops, I hope it's better now!